packages/driver/cypress/e2e/commands/cookies.cy.js

```diff
@@ -943,26 +972,55 @@ describe('src/cy/commands/cookies', () => {
     describe('.log', () => {
       beforeEach(function () {
         cy.on('log:added', (attrs, log) => {
-          if (attrs.name === 'getCookies') {
+          if (attrs.name === 'getAllCookies') {
```

Comment:
This suite was copied & pasted from getCookies log test and didn't get updated to verify getAllCookies. We accidentally were testing getCookies x2.